### ISSUE TITLE
Method argument autocomplete not working.?

### ISSUE THREAD

**Issue by xDGameStudios** [GodotNativeTools/godot-cpp#247]
Right now if I register a method that has arguments those don't appear in the autocomplete in the GDScript side... is it okay.. is it a bug?! Also, GDScript doesn't work with returned static typed instances from GDNative classes...

```
var a = MyGDNativeClass.new(); # "a" has autocompletion for methods

var b = a.return_an_instance_of_another_gdnative_class(); # "b" doesn't have autocompletion

var c: AnotherGDNativeClass = a.return_an_instance_of_another_gdnative_class(); # no autocompletion
```

**Comment by karroffel** [comment]
Yes, this is normal. That's a limitation in the GDScript autocompleter. It only checks other GDScript's and engine classes. Mono/C# and VisualScript behave in the same way (at least last time I checked).

**Comment by 3ddelano** [comment]
Any updates on this in Godot 3.4,
I'm facing same issue where Ref<MyNativeClass> doesn't get autocomplete in editor but MyNativeClass gets.

**Comment by Calinou** [comment]
> Any updates on this in Godot 3.4, I'm facing same issue where Ref doesn't get autocomplete in editor but MyNativeClass gets.

Could you upload a minimal reproduction project?

**Comment by 3ddelano** [comment]
Sure here is a demo showcasing this bug. https://github.com/3ddelano/ref-wrapped-autocomplete
